backend/app.py

```diff
@@ -492,13 +492,15 @@ def chatgpt():
     conversation_id = request.json["conversation_id"]
     question = request.json["query"]
     file_blob_url = request.json["url"]
+    agent = request.json.get("financial","consumer")
```

Comment:
Another thing, why is this written like this?

```python
agent = request.json.get("financial", "consumer")
```
Instead of this:

```python
agent = request.json.get("agent")
```